docs/molgenis/use_usingpyclient.md

```diff
@@ -0,0 +1,281 @@
+# Molgenis Pyclient
+The MOLGENIS EMX2 Python client allows the user to retrieve, create, update and delete entities on a MOLGENIS EMX2 server using scripts or programs written in the Python language.
+
+
+## Installation
+The releases of the package are hosted at [PyPI](https://pypi.org/project/molgenis-emx2-pyclient/).
+The recommended way to install the latest
+
+```commandline
+pip install molgenis_emx2_pyclient
+```
+
+## Setting up the client
+The Python client can be integrated in scripts authorized by either a username/password combination or a temporary token.
+
+Signing in with a username/password combination requires using the client as context manager:
+```python
+from molgenis_emx2_pyclient import Client
+
+username = 'username'
+password = '********'
+
+# Initialize the client as a context manager
+with Client('https://example.molgeniscloud.org') as client:
+    # Apply the 'signin' method with the username and password
+    client.signin(username, password)
+    
+    # Perform other tasks
+    ...
+```
+
+Before using the Pyclient with a token, this token should be generated in the UI, see [Tokens](use_tokens.md).
+Using the Pyclient with a token requires supplying the token in the initialization of the Client object:
+```python
+from molgenis_emx2_pyclient import Client
+
+token = '********************************'
+
+client = Client(url='https://example.molgeniscloud.org', token=token)
+    
+# Perform other tasks
+...
+
+```
+
+Additionally, if the Pyclient is to be used on a particular schema, this schema can be supplied in the initialization of the client, alongside the server URL:
+```python
+with Client('https://example.molgeniscloud.org', schema='My Schema') as client:
+    ...
+```
+or 
+```python
+client = Client('https://example.molgeniscloud.org', schema='My Schema', token=token)
+```
+
+## Methods and properties
+This section outlines some of the methods that are supported by the Pyclient.
+The results of these methods depend on the permission level the user has on the schemas involved in the method.
+
+The user roles on MOLGENIS EMX2 and their respective permissions are described in the [Permissions](use_permissions.md) section.
+
+
+### schema_names
+```python
+client.schema_names
+```
+
+Returns a list of the names of the schemas for which the user has at least _viewer_ permissions.
+
+### status
+```python
+client.status
+```
+The `status` property returns a string with information, including the server URL, the user (if applicable), the sign-in status, the version of MOLGENIS EMX2 running on the server, and the result of `schema_names`.
+
+
+### get_schemas
+```python
+client.get_schemas()
+```
+Retrieves the schemas for which the user has at least _viewer_ permissions as a list of dictionaries containing for each schema the id, name, label and description.
```

Comment:
For discussion later:
Do we want to return this data as list, why not a immutable tuple?

Reply:
I made `get_schema_metadata` private and hid its documenation from the README